### PR TITLE
convert module -> package

### PR DESCRIPTION
**Pull request by h4ck3rk3y**

### CODE REVIEW

**Comment by h4ck3rk3y** [comment]
Closes #19

File: types.proto [hunk 1]
```diff
@@ -16,15 +16,15 @@ message ModuleInput {
 	//  - only Ethereum network (EL and CL nodes) will be launched. Nothing else (no transaction spammer)
 	//  - params for the CL nodes will be ignored (e.g. CL node image, CL node extra params)
 	// This is a hack - it's not very elegant - but this is a commonly-requested feature
-	// The longterm solution is making the module trivial to decompose so we don't need flags like this; we're working
+	// The longterm solution is making the package trivial to decompose so we don't need flags like this; we're working
```
Comment (leoporoli) [inline types.proto:19]: should we rename the message name also `ModuleInput` ? or is this the expect name in APIC?

**Review by leoporoli — APPROVED** [review]

File: types.proto (hunk 1 above)
Comment (h4ck3rk3y) [reply types.proto:19]: Its set to that in APIC, Guillaume will be removing proto completely.

Comment (leoporoli) [reply types.proto:19]: ah! ok, thanks

**Pull request merged**